docs/production/releases.md

```diff
@@ -1,46 +1,70 @@
-## Releases
+# Releases
 
 Releases are automated by [this workflow](.github/workflows/030_create_release.yaml). When a release should be created following things need to be done:
 
-*   substrate-node
-    *   Increment spec version in the runtime [lib.rs](../../substrate-node/runtime/src/lib.rs)
-    *   Increment version in [Cargo.toml](../../substrate-node/Cargo.toml)
-    *   Increment chart `version` filed in [Chart.yaml](../../substrate-node/charts/substrate-node/Chart.yaml)
-    *   Increment chart `appVersion` filed in [Chart.yaml](../../substrate-node/charts/substrate-node/Chart.yaml)
+## Version bump
 
-*   tfchainbridge
-    *   Increment chart `version` filed in [Chart.yaml](../../bridge/tfchain_bridge/chart/tfchainbridge/Chart.yaml)
-    *   Increment chart `appVersion` filed in [Chart.yaml](../../bridge/tfchain_bridge/chart/tfchainbridge/Chart.yaml)
+This step can be done using Makefile or manually.
 
-*   activation-service
-    *   Increment chart `version` filed in [Chart.yaml](../../activation-service/helm/tfchainactivationservice/Chart.yaml)
-    *   Increment chart `appVersion` filed in [Chart.yaml](../../activation-service/helm/tfchainactivationservice/Chart.yaml)
-    *   Increment package `version` in [package.json](../../activation-service/package.json)
+### Makefile
 
-*   Js TFChain Client
-    *   Increment package `version` in [package.json](../../clients/tfchain-client-js/package.json)
+see makefile [here](../../Makefile)
 
-*   Scripts
-    *   Increment package `version` in [package.json](../../scripts/package.json)
+#### Usage
 
-*   Tools/fork-off-substrate
-    *   Increment package `version` in [package.json](../../tools/fork-off-substrate/package.json)
+```bash
+type=patch make version-bump # incrment the patch version
+type=minor make version-bump # incrment the minor version
+type=major make version-bump # incrment the major version
+```
 
-*   Commit the changes
+This function will take care also of branch and commit creation. Review the changes and push them, then follow the steps 3 and 4 below to finish the release.
 
-*   Create a new tag with the version number prefixed with a `v` (e.g. `v1.0.0`, `v1.0.0-rc1` for release candidates)
+Important: This function will also incrment the spec version in the runtime. If you already did this in another commit you need to undo spec_version changes to avoid double incrment.
 
-*   Push the tag to the repository
+### Manually
 
-*   The workflow will create a release draft with the changelog and the binaries attached
+1 - create the a new branch for the release. increment the version for all components in the monorepo. here is a list of the files that need to be changed to make the release:
```

Comment:
- Create a new branch for the release, increment the version for all components in the monorepo. Here is a list of the files that need to be changed to make the release: